validator/client/validator.go

```diff
@@ -277,6 +277,8 @@ func (v *validator) checkAndLogValidatorStatus(validatorStatuses []*ethpb.Valida
 			validatorActivated = true
 		case ethpb.ValidatorStatus_EXITED:
 			log.Info("Validator exited")
+		case ethpb.ValidatorStatus_INVALID:
+			log.Error("Invalid BLS signature")
```

Comment:
This log could be a bit confusing as a user, maybe it should say something like "Invalid Eth1 deposit"? Also, I don't think it should be an error, it's not necessarily an error.

Reply:
Hmm, are you suggesting Warning?
If a validator has an invalid Eth1 deposit, he shouldn't be able to operate right? Wouldn't "error" be appropriate here?

Reply:
Errors don't stop execution, and even if they have an invalid deposit, it doesn't mean it should stop the client. If a deposit is invalid their validator will never be activated anyways right?
This is mainly just to inform the user that the deposit is invalid.
I was thinking of it being an info log, because all this is doing is informing the user. But maybe it would be a good idea to be a warning to make sure the user notices the mistake. Wdyt @prestonvanloon ? Info or warning log from the validator client?